utils/maven-plugin/src/main/java/io/apicurio/registry/maven/DownloadRegistryMojo.java

```diff
@@ -117,17 +116,13 @@ protected void executeInternal() throws MojoExecutionException {
 
             try {
                 Integer version = versions.get(id);
-                Response response = (version != null) ?
-                                    getClient().getArtifactVersion(version, id) :
-                                    getClient().getLatestArtifact(id);
-                try (InputStream stream = response.readEntity(InputStream.class)) {
-                    if (replaceExisting) {
-                        Files.copy(stream, outputFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
-                    } else {
-                        Files.copy(stream, outputFile.toPath());
-                    }
-                } finally {
-                    response.close();
+                InputStream stream = (version != null) ?
+                        getClient().getArtifactVersion(version, id) :
+                        getClient().getLatestArtifact(id);
+                if (replaceExisting) {
+                    Files.copy(stream, outputFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
+                } else {
+                    Files.copy(stream, outputFile.toPath());
```

Comment:
Can you confirm that `Files.copy()` closes the stream?

Reply:
Good catch, no, it does not close the stream.